spec/lib/rails-settings-ui/value_types/array_spec.rb

```diff
@@ -11,5 +11,11 @@
       array_type = RailsSettingsUi::ValueTypes::Array.new("auto")
       expect(array_type.cast).to eq(:auto)
     end
+
+    it "should accept ActionController::Parameters" do
+      parameter = ActionController::Parameters.new('stuff' => 'on', 'other'=> 'off')
```

Comment:
Line is too long. [84/80]<br>Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.